lib/enumerize/activerecord.rb

```diff
@@ -59,5 +65,19 @@ def update_all(updates)
         super(updates)
       end
     end
+
+    class Type < ActiveRecord::Type::Value
```

Comment:
@lest I reproduce it: https://github.com/brainspec/enumerize/issues/272#issuecomment-294330168